src/main/java/lotto/domain/LottoTicket.java

```diff
@@ -0,0 +1,58 @@
+package lotto.domain;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class LottoTicket {
+
+    public static final Money PRICE = Money.valueOf(1000);
+
+    public static final int LOTTO_NUMBER_COUNT = 6;
+
+    private final List<LottoNumber> lottoNumbers;
+
+    private LottoTicket(final List<LottoNumber> lottoNumbers) {
+        this.lottoNumbers = Collections.unmodifiableList(lottoNumbers);
+    }
+
+    public static LottoTicket of(final String... lottoNumbers) {
+        return Arrays.stream(lottoNumbers)
+                .map(LottoNumber::valueOf)
+                .collect(Collectors.collectingAndThen(Collectors.toList(), LottoTicket::of));
+    }
+
+    public static LottoTicket of(final int... lottoNumbers) {
+        return Arrays.stream(lottoNumbers)
+                .mapToObj(LottoNumber::valueOf)
+                .collect(Collectors.collectingAndThen(Collectors.toList(), LottoTicket::of));
+    }
+
+    public static LottoTicket of(final List<LottoNumber> lottoNumbers) {
+        HashSet<LottoNumber> removedDuplication = new HashSet<>(lottoNumbers);
```

Comment:
사실 반환 타입이나, 인스턴스 변수 선언부가 아니라 크게 상관은 없을 것 같지만..

다형성의 관점에서 아래와 같이 선언하고 사용하시면 어떨까요?

```suggestion
        Set<LottoNumber> removedDuplication = new HashSet<>(lottoNumbers);
```

똑같이 동작하니 상관없지 않냐고 하시면 크게 할말은 없지만.. Set 키워드가 더 짧기도 하고 다형성의 관점에서 위와 같이 선언해주시면 좋을 것 같습니다!

Reply:
저도 규남님 의견에 동의합니다! 수정하도록 하겠습니다 :)